ui/pages/settings/security-tab/metametrics-toggle/index.ts

```diff
@@ -0,0 +1 @@
+export { default } from './metametrics-toggle';
```

Comment:
Ugh barrel files... I don't see a point in this, but if it is a pattern then fine.